pages/test_analytics/flaky_test_assignment.md

```diff
@@ -11,15 +11,21 @@ To enable assignments, you must have at least one team that has access to your s
 
 ## Assigning a test
 
-When viewing your flaky tests, you should now see a list of teams with suite access permissions listed inside the assignment dropdown. From here you may assign, reassign or remove the assignment.
+When viewing your flaky tests, you should now see a list of teams with suite access permissions listed inside the manage flaky dropdown. From here you may assign, reassign or remove the assignment.
```

Comment:
Maybe "flaky management dropdown"? I think this might read weirdly to a non TA person?